app/src/sharedTest/java/org/oppia/android/app/resumelesson/ResumeLessonFragmentTest.kt

```diff
@@ -119,7 +121,7 @@ class ResumeLessonFragmentTest {
   private fun setUpTestApplicationComponent() {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
-
+  @Ignore
```

Comment:
Newline before this (newlines generally always separate class members except fields which are optional).

Reply:
ignore is not required, this test is for tablets and it fails on mobile, so I added this while testing the tests on espresso

Reply:
removed ignore